src/sidebars/sidebar.html

```diff
@@ -16,6 +16,20 @@
                 </div>
             </div>
         </div>
+        <div ng-controller="ClosedWindowsCtrl as closedWindowsCtrl">
+            <div class="favClosed">
+                <icon name="closed_tabs" ng-show="true" icon-click="closedTabsShow = true"
+                      ng-click="$event.stopPropagation(); closedWindowsCtrl.refreshClosedWindows()"></icon>
```

Comment:
I notice both `icon-click` and `ng-click` are being used here. Is there a reason for using both, or can they be combined?